app/static/css/style.css

```diff
@@ -115,35 +153,119 @@ footer {
 }
 
 .thin-footer {
-  font-family:'Nunito-Sans', sans-serif;
+  font-family:'Inter', sans-serif;
   font-weight: 100;
   letter-spacing: 0.5px;
   color: var(--grey-mid);
 }
 
+.expand {
+  transition: all ease 0.5s;
+}
+
+.expand:hover {
+  font-size: 101%;
+  color: var(--grid-grey);
+}
+
 
 
 /**
- * Front page banner
+ * Front page strips
  */
 
-#splash-img {
+ .animate {
+  animation : move 7s infinite;
+ }
+
+ @keyframes move {
+  0%, 100% {
+    transform : translate(5px, 5px);
+  }
+  50% {
+    transform: translate(-5px, -5px);
+  }
+}
+
+.animate2 {
+  animation : move2 7s infinite;
+ }
+
+ @keyframes move2 {
+  0%, 100% {
+    transform : translate(5px, -5px);
+  }
+  50% {
+    transform: translate(-5px, 5px);
+  }
+}
+
+.animate3 {
+  animation : move3 7s infinite;
+ }
+
+ @keyframes move3 {
+  0%, 100% {
+    transform : translate(5px, 9px);
+  }
+  50% {
+    transform: translate(-5px, 9px);
+  }
+}
```

Comment:
There has to be a better way to do this.

Reply:
what would you suggest?